### PR TITLE
Added Option to change font style of whole system

### PR DESCRIPTION
**Pull request by SJPBP**
Users can specify font style and weight, such as using the wholesystem-font-weight property, to control the appearance of text.

### CODE REVIEW

**Comment by Spartanlasergun** [comment]
Can't merge. Your variable names can't have hyphens in python. Also, when you pass the variable as a parameter, don't put the variable name in quotes; you're passing a string not the variable itself. Other than that good job. (Please test to ensure that your code runs before you submit another pull request)

File: src/calendar_widget/Calendar_Widget.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self, window_name, pos_x=0, pos_y=0, size=300, style=None, command=
                  calendar_date_title="black", date_text_fill="black", trail_box_fill="gray83", trail_text_fill="black",
                  date_highlight="orange", text_highlight_fill="black", weekday_font_family="Algerian",
                  date_heading_font_family="Garamond", date_text_font_family="Arial CE", user_highlight_colour="gray75",
-                 user_highlight_text="black"):
+                 user_highlight_text="black", wholesystem-font-weight="normal", wholesystem-font-style="roman"):
```
Comment (Spartanlasergun) [inline src/calendar_widget/Calendar_Widget.py:21]: remove hypens. python variables can't have hyphens in them

File: src/calendar_widget/Calendar_Widget.py [hunk 2]
```diff
@@ -136,7 +136,7 @@ def __init__(self, window_name, pos_x=0, pos_y=0, size=300, style=None, command=
             heading_inc = heading_inc + 1
 
         font_size = int((box_depth * 0.75) * 0.75)
-        weekday_font = font.Font(family=weekday_font_family, size=font_size)
+        weekday_font = font.Font(family=weekday_font_family, size=font_size, weight="wholesystem-font-weight", slant="wholesystem-font-style")
```
Comment (Spartanlasergun) [inline src/calendar_widget/Calendar_Widget.py:139]: You cant pass the variable if you use quotes.
weight="wholesystem_font_weight" ---incorrect 
weight=wholesystem_font_weight    ---correct

**Review by Spartanlasergun — COMMENTED** [review]
reviewed and closed

**Comment by SJPBP** [comment]
Thank you for telling me about the error

**Pull request closed without merging**